Arduboy.cpp

```diff
@@ -682,14 +687,12 @@ unsigned char* Arduboy::getBuffer()
 
 boolean Arduboy::pressed(uint8_t buttons)
 {
- uint8_t button_state = getInput();
- return (button_state & buttons) == buttons;
```

Comment:
>  I tend to avoid defining local variables that are only used once, 

Just so you're aware though there isn't any type of inefficiency in this since the compiler knows it's only used once - the compiled code probably is exactly the same - since the compiler needs a variable/register to put the result in anyways whether you ask for one or not.